underfs/swift/pom.xml

```diff
@@ -81,7 +51,13 @@
     <dependency>
       <groupId>org.javaswift</groupId>
       <artifactId>joss</artifactId>
-      <version>0.9.8</version>
+      <version>0.9.10</version>
+      <exclusions>
+          <exclusion>
+              <groupId>org.slf4j</groupId>
+              <artifactId>slf4j-api</artifactId>
+          </exclusion>
```

Comment:
@calvinjia Without this exclusion i saw collisions in Spark. Seems collide with some other package. Not sure it's present in all versions of Spark. But in my case, there were warnings and exceptions from Spark, complaining on collision with this package. Exclusion resolved it.